PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD003.xml

```diff
@@ -0,0 +1,231 @@
+                                        </item>
+                                        <item xml:id="q5" n="5">
+                                           <dim unit="leaf">8</dim>
+                                            <locus from="33r" to="40v"/>
+                                        </item>
+                                        <item xml:id="q6" n="6">
+                                          <dim unit="leaf">8</dim>
+                                            <locus from="41r" to="48v"/>
+                                        </item>
+                                        <item xml:id="q7" n="7">
+                                          <dim unit="leaf">8</dim>
+                                            <locus from="49r" to="56v"/>
+                                        </item>
+                                        <item xml:id="q8" n="8">
+                                         <dim unit="leaf">6</dim>
+                                            <locus from="57r" to="62v"/>
+                                        </item>
+                                       </list>
+                                </collation>
+                                <condition key="good">The beginning and end of <ref target="#ms_i1"></ref> and the binding are missing. 
+                                    The other text block is in a good condition.</condition>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="12">
+                                     <ab type="pricking">Primary pricks are sometimes visible, horizontal pricks are not visible.</ab>
+                                    <ab type="ruling">Ruling is visible.</ab>
+                                    <ab type="ruling">The upper line is written above the ruling,
+                                        the lower line is written above the ruling.</ab>
+                                    <locus from="1r" to="62r"/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>The manuscript can be dated to the fourteenth or fifteenth century based on palaeographic features. For example, the letter መ shows a tendency for loop separations, ሎ demonstrates the
+                                    vowel marker with a linking line, ጵ exhibits the form used before the sixteenth century. Additionally, the word እግዚአብሔር is 
+                                    written as a single word. Overall, the manuscript displays the standard orthography 
+                                    of medieval Ethiopic. </desc>
+                               <seg type="ink">Black, red.</seg>
+                                <date notBefore="1400" notAfter="1500"> </date>
```

Comment:
remove space between `>` and `<` or the space will be printed instead of the date